README.md

```diff
@@ -25,9 +26,9 @@ Recommandation](http://www.ivoa.net/documents/latest/UCDlistMaintenance.html).
 ## Status?
 
 The last stable version is
-**[REC-1.3](http://www.ivoa.net/documents/UCD1+/20180527/index.html)**.
+**[EN-1.4](https://ivoa.net/documents/UCD1+/20210430/index.htmll)**.
```

Comment:
I think the date should reflect the endorsement, i.e. 20210616, before it was in Proposed EN status and the status change means also a date one (that can be reflected in the Makefile).

Reply:
ok, let me close this PR (and the 2 other that are pending) and I'll do a complete PR from my fork with those changes.